google_gemini/README.md

```diff
@@ -0,0 +1,125 @@
+# Google Gemini
+
+## Overview
+
+Monitor, troubleshoot, and evaluate your LLM-powered applications, such as chatbots or data extraction tools, using [Google Gemini](https://gemini.google.com/).
+
+If you are building LLM applications, use LLM Observability to investigate the root cause of issues, monitor operational performance, and evaluate the quality, privacy, and safety of your LLM applications.
+
+[LLM Obs tracing view video](https://imgix.datadoghq.com/video/products/llm-observability/expedite-troubleshooting.mp4?fm=webm&fit=max )
+
+## Setup
+
+### LLM Observability: Get end-to-end visibility into your LLM application using Google Gemini
+You can enable LLM Observability in different environments. Follow the appropriate setup based on your scenario:
+
+#### Installation for Python
+
+##### If you do not have the Datadog Agent:
+1. Install the `ddtrace` package:
+
+  ```shell
+    pip install ddtrace
+  ```
+
+2. Start your application with the following command, enabling Agentless mode:
+
+  ```shell
+    DD_SITE=<YOUR_DATADOG_SITE> DD_API_KEY=<YOUR_API_KEY> DD_LLMOBS_ENABLED=1 DD_LLMOBS_AGENTLESS_ENABLED=1 DD_LLMOBS_ML_APP=<YOUR_ML_APP_NAME> ddtrace-run python <YOUR_APP>.py
+  ```
+
+##### If you already have the Datadog Agent installed:
+1. Make sure the Agent is running and that APM and StatsD are enabled. For example, use the following command with Docker:
+
+  ```shell
+  docker run -d \
+    --cgroupns host \
+    --pid host \
+    -v /var/run/docker.sock:/var/run/docker.sock:ro \
+    -v /proc/:/host/proc/:ro \
+    -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+    -e DD_API_KEY=<DATADOG_API_KEY> \
+    -p 127.0.0.1:8126:8126/tcp \
+    -p 127.0.0.1:8125:8125/udp \
+    -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true \
+    -e DD_APM_ENABLED=true \
+    gcr.io/datadoghq/agent:latest
+  ```
+
+2. If you haven't already, install the `ddtrace` package:
+
+  ```shell
+    pip install ddtrace
+  ```
+
+3. Start your application using the `ddtrace-run` command to automatically enable tracing:
+
+  ```shell
+     DD_SITE=<YOUR_DATADOG_SITE> DD_API_KEY=<YOUR_API_KEY> DD_LLMOBS_ENABLED=1 DD_LLMOBS_ML_APP=<YOUR_ML_APP_NAME> ddtrace-run python <YOUR_APP>.py
+  ```
+
+**Note**: If the Agent is running on a custom host or port, set `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` accordingly.
+
+##### If you are running LLM Observability in a serverless environment (AWS Lambda):
```

Comment:
```suggestion
##### If you are running LLM Observability in a serverless environment :
- Example for AWS Lambda :
```

Reply:
@buraizu would this suffice ? If yes feel free to commit this change 🙏

Reply:
Thanks, I edited the phrasing in your suggestion slightly. Does this look alright to you?